src/hooks/useAudio/getDevices.ts

```diff
@@ -1,8 +1,9 @@
 async function getDevices(type?: MediaDeviceKind): Promise<MediaDeviceInfo[]> {
```

Comment:
Acho interessante mudar esse nome "MediaDeviceKind" para "MediaDeviceType". Nesse sentido "kind" não parece tão certo, ainda mais porque o nome da variável é "type". [Vê isso aqui](https://askanydifference.com/difference-between-kind-and-type/#:~:text=Type%20is%20used%20to%20show%20category%20or%20a%20sub%2Ddivision,a%20bigger%20or%20whole%20thing.&text=Kind%20can't%20be%20used,to%20a%20person%20or%20something.&text=Kind%20is%20more%20preferred%20in%20speeches%20than%20writing.)

Reply:
`MediaDeviceKind` é um tipo global do TS em lib.dom.d.ts, ao menos na versão 4.5.4